cob_twist_controller/src/cob_twist_controller.cpp

```diff
@@ -37,46 +37,33 @@
 
 #include "cob_obstacle_distance/Registration.h"
 
-#define DEBUG_BASE_ACTIVE    0
-#define DEBUG_BASE_COMP     0
+#define MESH_RES_MARKER 99u // Must be equal to "cob_obstacle_distance/obstacle_distance_data_types.hpp"
```

Comment:
how can we assure this to be equal?
could we not keep to the defines in http://docs.ros.org/api/visualization_msgs/html/msg/Marker.html?
There is a `uint8 MESH_RESOURCE=10` and you have been using the `uint8 SPHERE=2` before...see [here](https://github.com/ipa-fxm/cob_control/pull/7/files#diff-b2d7bf442cb46be9174e8e9246ee1955R536)